haiku-apps/deeperpeople/deeperpeople-1.0.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A contacts application"
+DESCRIPTION="DeeperPeople, a contacts application for Haiku."
+APP="DeeperPeople"
+HOMEPAGE="https://github.com/HaikuArchives/$APP"
+COPYRIGHT="1991-99 Olivier Cortes"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="9db5a5a969814156c0df01f7bbc5813438464498"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="41a18e6245114172771aaf282a334d1e92edf494d38b0e38dcdf96c11b561d59"
+SOURCE_DIR="$APP-$srcGitRev"
```

Comment:
I'll try to make some suggestions (@fbrosson or others will give you better ones, though):
I noticed this has a "src" directory inside the source.
I would rename the recipe as `deeperpeople-1.0.0~git.recipe`, and add:
```
SOURCE_FILENAME="$APP-$portVersion-$srcGitRev.tar.gz"
SOURCE_DIR="$APP-$srcGitRev"
```